context/on_startup.py

```diff
@@ -1,18 +1,24 @@
-import json
-import requests
 import SimpleHTTPServer
 import SocketServer
+import json
+import os
+import requests
+
 from cgi import escape
 
 
+def get_refinery_input():
+    """ Make a GET request to acquire the input data for the container"""
+    return requests.get(os.environ["INPUT_JSON_URL"]).json()
```

Comment:
But isn't refinery using using json in `INPUT_JSON`? I could be confused.

Reply:
INPUT_JSON_URL is what refinery is passing along to containers after: https://github.com/refinery-platform/refinery-platform/pull/2677